Makefile

```diff
@@ -142,17 +142,26 @@ cleanup-test-cockroach:
 	@docker rm dss-crdb-for-testing > /dev/null 2>&1 || true
 
 .PHONY: test-e2e
-test-e2e:
-	test/docker_e2e.sh
```

Comment:
One regression in this PR is that docker-compose invokes a build for each of `local-dss-rid-bootstrapper`, `local-dss-scd-bootstrapper`, `local-dss-grpc-backend`, and `local-dss-http-gateway`.  It does (already) manage rebuilding correctly, using the cache for everything if nothing has changed.  However, just invoking the build command (even when using the cache for everything) takes 14-18 seconds on my machine which means doubling the bootstrapper build and doubling the main image build adds 30+ seconds to the e2e invocation cycle versus the current docker_e2e.sh.  Since the e2e test cycle is a primary development tool, I think it's worth trying to just have each of those images build once.  I've added a commit that fixes this (no more action required).